File: DVG.CRM.XeCung.ClientSide/dvg-crm-xecung-final-client/src/app/models/video-index-model.ts
import { Paging } from './pagingmodel';

export class VideoIndexModel extends Paging {
    constructor() {
        super();
        this.VideoType = 0;
        this.SpendDate = null;
    }
    FilterKeyWord: string;
    VideoType: number;
    SpendDate: string;
}

File: DVG.CRM.XeCung.ApplicationLayer/AppServices/Videos/Models/VideoIndexModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DVG.CRM.XeCung.ApplicationLayer.AppServices.Videos.Models
{
    public class VideoIndexModel
    {
        public string FilterKeyword { get; set; }
        public int VideoType { get; set; }
        public System.DateTime? SpendDate { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }

    }
}
